k8s/distributed-chroma/templates/coordinator.yaml

```diff
@@ -24,19 +25,47 @@ spec:
           imagePullPolicy: IfNotPresent
           name: coordinator
           ports:
-            - containerPort: 50051
+            - containerPort: {{ .Values.coordinator.port }}
```

Comment:
maybe I would do port not configurable at the container lvl, because it will also require code to listen the port. But I would do the service port configurable?

Reply:
Good call